examples/sveltekit-1/src/routes/+layout.svelte

```diff
@@ -1,6 +1,20 @@
 <script>
+	import { browser } from '$app/environment';
+	import { page } from '$app/stores';
+	import { webVitals } from '$lib/vitals';
 	import Header from './Header.svelte';
 	import './styles.css';
+
+	/** @type {import('./$types').LayoutServerData} */
+	export let data;
+
+	if (browser && data?.analyticsId) {
```

Comment:
I just realized that you probably want to rerun this on every page change (in the client), not just on first page load - in that case the statement needs to be reactive to fire on changes. [the old example](https://github.com/vercel/vercel/blob/main/examples/sveltekit/src/routes/%2Blayout.svelte#LL10C3-L10C6) also does it this way
```suggestion
	$: if (browser && data?.analyticsId) {
```